ci/azure-pipelines-dev.yml

```
@@ -79,6 +79,7 @@ stages:
                 setup.py
                 MANIFEST.in
                 tests/**
+                docs/*.py
```

I think we want recursive here:
```suggestion
                docs/**/*.py
```